Changelog v1.8.x.md

```diff
@@ -14,6 +14,8 @@
 - New: A briefcase's owner can now be changed if its owner rightclicks while holding the briefcase in their off hand and a named Universal Owner Changer in their main hand (Thanks Redstone_Dubstep!)
 - New: The codebreaker can now be used on a briefcase by holding the briefcase in the off hand and the codebreaker in the main hand and rightclicking (Thanks Redstone_Dubstep!)
 - New: Customization option to change how long the Keycard Reader emits a redstone signal when it has been activated (Thanks Redstone_Dubstep!)
+- New: Secret Signs can now have a whitelist (Thanks Redstone_Dubstep!)
+- New: New customization option to make the text of Secret Signs visible to everyone (Thanks Redstone_Dubstep!)```

```

Comment:
There are three backticks at the end here. Please excuse this, GitHub's formatter does not see three backticks that are not on a seperate line as the end to a code block. I have edited my previous main review comment to fix it. Please check it again and verify you've done everything I noted there.